misc/db_schema.yaml

```diff
@@ -2057,8 +2058,8 @@ syslog:
     - { Field: device_id, Type: 'int unsigned', 'Null': true, Extra: '' }
     - { Field: facility, Type: varchar(10), 'Null': true, Extra: '' }
     - { Field: priority, Type: varchar(10), 'Null': true, Extra: '' }
-    - { Field: level, Type: varchar(10), 'Null': true, Extra: '' }
```

Comment:
seems that you have something on the syslog side that should not be part of this PR

Reply:
yeah. I noticed it by myself. thanks